canister/src/main.rs

```diff
@@ -76,6 +76,14 @@ pub fn bitcoin_get_utxos_query(request: GetUtxosRequest) {
     };
 }
 
+#[update(manual_reply = true)]
+pub fn bitcoin_get_block_headers(request: GetBlockHeadersRequest) {
```

Comment:
I'm wondering if this is a good opportunity to start using [PocketIC](https://crates.io/crates/pocket-ic). Perhaps we can have a test where we query this endpoint and verify that right now it traps with a message saying that it's unimplemented, and later that test can evolve as you build the functionality.

Reply:
How about I add tests with PocketIC in the follow-up?